apps/balancer-tools/src/lib/balancer/gauges.ts

```diff
@@ -3,7 +3,7 @@ import { NetworkChainId } from "@bleu-balancer-tools/utils";
 import POOLS_WITH_GAUGES from "#/data/voting-gauges.json";
 
 export const POOLS_WITH_LIVE_GAUGES = POOLS_WITH_GAUGES.filter(
-  (pool) => !pool.gauge.isKilled && pool.gauge.addedTimestamp,
```

Comment:
pools whose gauge doesn't have a timestamp have a different type of gauge and shouldn't be accounted for